Jenkinsfile

```diff
@@ -0,0 +1,363 @@
+#!groovy
+
+def xl_LatestVersion = ""
+
+pipeline {
+    agent none
+
+    parameters {
+        choice(
+            name: 'XLProduct',
+            choices: ['xl-release', 'xl-deploy'],
```

Comment:
why not running the pipeline for both products at the same time? is there a limitation on this

Reply:
Nope .. so i am updating it to support both products .. the only thing is that we can not render both at same command as we can only pass one xl-version and during pull requests we are getting latest ones which are not always the same version for both products hence i am going to do this on 2 rendering steps